pkg/v1/cli/command/core/context.go

```diff
@@ -0,0 +1,643 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
```

Comment:
Planning to add `context_test.go`?

Reply:
Was part of the other PR. Have removed those elements.